stories/InfiniteHits.stories.js

```diff
@@ -1,5 +1,6 @@
 import { storiesOf } from '@storybook/vue';
 import { action } from '@storybook/addon-actions';
+import { createInfiniteHitsSessionStorageCache } from 'instantsearch.js/es/lib/infiniteHitsCache';
```

Comment:
Should we reexport `createInfiniteHitsSessionStorageCache` from `vue-instantsearch`?

Reply:
It's an interesting idea, but we don't do that for now for other things like router, I'm not sure if that can have a bundle size impact?

Reply:
I don't think it'll have a bundle size impact but it's worth trying.

On a UX perspective, it doesn't make sense to import something from a library's dependency (you're not even aware that you use `instantsearch.js` when you use `vue-instantsearch`).

Reply:
let's do it in a stacked PR? There's argument for exposing the connectors etc. too, so let's join that?

Reply:
I guess not in a stacked PR but a separate PR as in something like `chore: reexport things from IS.js...`

If we're going to reexport stuff, then we'd better do them altogether with doc changes.
What do you think?